Change remember period to range slider

subdavis opened the pull request:
Here's a small change I made to see how #156 might be done.   Haven't bothered styling it yet, but it's totally functional.

Slider all the way left: don't remember
Slider all the way right: remember forever
Anything in the middle: I have this going from 2 to 24 hours, but it could be any range.

![screenshot](http://i.imgur.com/dgp2qy8.png)

suBDavis commented:
![final](https://cloud.githubusercontent.com/assets/4214172/25777679/43608e46-32b2-11e7-8b68-b0f7e65171fa.png)

I think this one is ready to go - styled the slider to match the unlock button and the general feel of the ui.

DukeyToo commented:
It would be great if it remembered the original "remember" value before the slider was in place - that would make it a better transition for people already using one of the old button-based remember options.

For example, on first use, my remember slider reverted to "Do not remember" - the actual value was 30 (minutes), which corresponds with the new .5 value.

suBDavis commented:
Agreed.  Changed internal rememberPeriod value back to using minutes, and updated my new lookup table.  Changes are compatible between master and this feature now.

suBDavis commented:
I agree.  Reverted back to using minutes for rememberPeriod for compatibility.

DukeyToo commented on popups/controllers/masterPasswordController.js:
```diff
@@ -131,7 +166,8 @@ function MasterPasswordController($scope, $routeParams, $location, keepass, unlo
 
       if ($scope.rememberPeriod) {
-      	settings.setForgetTime('forgetPassword', (Date.now() + (60000*$scope.rememberPeriod)))
+        var check_time = 60000*60*$scope.rememberPeriod; // min per hour * milliseconds per min
```
I think this is still using the "hour" unit of measurement, i.e. it is multiplying by an extra 60?

subdavis commented:
Damn, it's like amateur hour over here.

Fixed and squashed everything down, since some of them were broken and this should be an all-or-nothing change.

DukeyToo commented:
Thanks much!
